pkgs/top-level/dotnet-packages.nix

```diff
@@ -220,6 +220,37 @@ let self = dotnetPackages // overrides; dotnetPackages = with self; {
 
   # SOURCE PACKAGES
 
+  Boogie = buildDotnetPackage rec {
+    baseName = "Boogie";
```

Comment:
If this is not an official release then doesn't the [package naming section](https://nixos.org/nixpkgs/manual/#sec-package-naming) apply? I.e., the name should be "Boogie-unstable" and the version "2017-01-03". I don't know if this convention apply for dotNET packages, though.